data/schema.js

```diff
@@ -1,4 +1,4 @@
-var getbabelRelayPlugin = require('babel-relay-plugin');
-var schema = require('./schema.json');
+var getbabelRelayPlugin = require('babel-relay-plugin')
+var schema = require('./schema.json')
 
-module.exports = getbabelRelayPlugin(schema.data);
\ No newline at end of file
+module.exports = getbabelRelayPlugin(schema.data, { substituteVariables: true })
```

Comment:
I thought this wasn’t really needed after all? I’d also nonetheless like a link to some doc or what have you about this.